EventFilter/GEMRawToDigi/BuildFile.xml

```diff
@@ -1,15 +1,3 @@
-<use name="FWCore/Framework"/>
-<use name="DataFormats/FEDRawData"/>
-<use name="FWCore/PluginManager"/>
-<use name="FWCore/ParameterSet"/>
-<use name="CondFormats/DataRecord"/>
-<use name="CondFormats/GEMObjects"/>
-<use name="FWCore/MessageLogger"/>
-<use name="Utilities/StorageFactory"/>
-<use name="FWCore/Sources"/>
-<use name="DataFormats/Common"/>
```

Comment:
this was added recently in 477a01203caeebd22d174ac02e90d5600c9cbd1e
it looks like the include of DataFormats/Common/interface/Handle.h is actually in EventFilter/GEMRawToDigi/plugins
The other two deps (MuonData and MuonDetId are apparently no longer needed, OK there).

I guess some other package carries the DataFormats/Common to keep things working.